src/store/getters.ts

```diff
@@ -239,17 +239,22 @@ export default {
             let assetsMarketCap: CurrenciesInfo = {} as any;
             let hasFiat = false;
             let hasTokenBalance = false;
+            let nativeAssetMarketCap = new BN(0);
 
             const fiatBalances = Object.entries(account.balances).reduce((accum, [asset, balance]) => {
               const fiat = assetFiatBalance(asset, new BN(balance));
               const marketCap = assetMarketCap(asset);
               const tokenBalance = account.balances[asset];
-              const { type } = cryptoassets[asset];
+              const { type, matchingAsset } = cryptoassets[asset];
 
               if (fiat) {
                 hasFiat = true;
                 assetsWithFiat.push({ asset, type, amount: fiat });
               } else if (marketCap) {
+                if (type === 'native' && !matchingAsset) {
```

Comment:
you can use the enum here instead of string `native`

Reply:
rdy